packages/icons/package.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "@clr/icons",
   "private": true,
-  "version": "5.0.3",
+  "version": "0.0.0",
   "scripts": {
     "build": "npm-run-all build:css build:optimize build:web build:package build:update",
     "build:preinstall": "mkdir -p ../../dist/clr-icons/ && cp ./package.json ../../dist/clr-icons/",
     "build:update": "node ../../scripts/update-local-packages.js /packages/icons/dist/clr-icons/package.json",
     "build:web": "webpack --config webpack.config.js",
     "build:css": "sass --precision=8 src/clr-icons.scss ./dist/clr-icons/clr-icons.css",
     "build:optimize": "csso -i ./dist/clr-icons/clr-icons.css -o ./dist/clr-icons/clr-icons.min.css -s file --no-restructure;",
-    "build:package": "cpy npm.json README.md ./dist/clr-icons/; mv ./dist/clr-icons/npm.json ./dist/clr-icons/package.json; replace '@VERSION' $npm_package_version ./dist/clr-icons/package.json; ",
+    "build:package": "cpy npm.json README.md ./dist/clr-icons/; mv ./dist/clr-icons/npm.json ./dist/clr-icons/package.json;",
```

Comment:
Updated to be consistent with the rest of the repo and use the version copy script